### ISSUE TITLE
Visual Bug with Better Combat [1.20.1]

### ISSUE THREAD

**Issue by Exokyhh** [Jeryn99/MC-Gliders#59]
When this mod is installed with **Better Combat** _(bettercombat-forge-1.8.4+1.20.1.jar)_, pressing the left mouse button while gliding results in the animation still occurring (shouldn't happen). Every weapon that uses the Better Combat animations is affected. 

I'm using:
Forge - 47.2.20
bettercombat-forge-1.8.4+1.20.1.jar
vc_gliders-forge-1.1.5.jar
simplyswords-forge-1.54.0-1.20.1.jar (not relevant for the issue but used it to test)

The video is enough to describe the problem.

https://github.com/Jeryn99/MC-Gliders/assets/90096585/def5f7ce-0353-4895-835d-0d72372000fb

**Comment by Jeryn99** [comment]
![image](https://github.com/Jeryn99/MC-Gliders/assets/36424503/65061ca0-2848-43be-b314-41212a2c6a3b)

There really needs to be someone stepping in and creating a standard for player animations...


I will look into this, but no promises, seems I need to do this for multiple mods now 


https://github.com/Jeryn99/MC-Gliders/issues

**Comment by Exokyhh** [comment]
I agree! It's very messy, and everyone does it differently. Now, even with the loaders, the Fabric split was not enough! Now NeoForge and Quilt exist. I wonder how better it could be if there were some standards and people used the same libraries, loaders, etc...
btw, I really like your mod. I've always loved the idea of having a paraglider in Minecraft, but basically the only choice I had was this mod (https://www.curseforge.com/minecraft/mc-mods/paragliders), which I've used a lot but I don't fully appreciate it, it's an old mod, it doesn't equip in a curios slot, it has no animations, and it replicates 1:1 the stamina system with Zelda. It's cool, but I don't see it as a good feature to see in Minecraft. Your mod is straightforward. It adds gliders, with tiers/upgrades, and with a cool animation too. No more, no less, just perfect :)

**Comment by Jeryn99** [comment]
I appreciate your comments about my mod, but the mod loader split was a good thing for the eco-system
Sure it split the player base but it made people move forward with better innovations for mods

**Comment by Exokyhh** [comment]
I've always watched it from a user perspective, and hated that each loader had their own really good mods but couldn't combine them together, but yeah, I guess it was a good thing. Maybe in the future I will look a bit into mod developing (since I've already learned Java) and check out the fabric and forge loaders; might as well enjoy it :)
